docs/book/src/developer/providers/migrations/v1.4-to-v1.5.md

```diff
@@ -20,6 +20,7 @@ maintainers of providers and consumers of our Go API.
 ### Deprecation
 
 - API version`v1alpha4` is deprecated and CAPI will stop serving this version in v1.6.
+- `sigs.k8s.io/cluster-api/controllers/remote.DefaultIndexex` has been deprecated and will be removed in a future release. Please use `sigs.k8s.io/cluster-api/controllers/external.NodeProviderIDIndex` instead.
```

Comment:
Should we add something like, but only if you are actually using the index?

I know it's a bit trivial but I can imagine folks are just blindly adding this index when migrating

Reply:
Sounds good!